docs/client.rst

```diff
@@ -430,3 +430,17 @@ the C++ example application at
 and a Python version at
 `src/clients/python/simple\_sequence\_client.py
 <https://github.com/NVIDIA/tensorrt-inference-server/blob/master/src/clients/python/simple_sequence_client.py>`_.
+
+Shape Tensor I/O
+^^^^^^^^^^^^^^^^
+
+TensorRT supports network I/O tensors that can be used to specify shape
+values. TRTIS allows users to deploy models with shape tensor I/O with
+a few restrictions if the batching is enabled. Firstly, client must
```

Comment:
"TensorRT models allow shape tensors. The inference server client libraries support these tensors using the existing APIs. When setting the value for a shape tensor only a single batch-1 value should be provided, even for a batch request. This single shape tensor is used for the entire batch but the batch dimension should not be included."

I would drop "Secondly..." since that should be covered in model configuration doc.